### DISCUSSION TITLE
blog首页的大卡片该如何个性化

### DISCUSSION THREAD

**Discussion by Yesord** [anzhiyu-c/hexo-theme-anzhiyu#294] [Q&A]
<img width="1425" alt="image" src="https://github.com/user-attachments/assets/64a5d5e2-700d-4e60-a6a8-4e2e3b61b946">
我想问问大佬，blog首页的两张大卡片该如何个性化呢
<img width="1473" alt="image" src="https://github.com/user-attachments/assets/2cf0b8e2-3119-412a-859a-eac07cab8be3">
我查网页代码可以看到好像是用到了todaycard这个东西，在public的文件里能找到相关的一些代码，但是试了一下好像不能在主题yml文件_config.anzhiyu.yml中改，本人不是做网页的，查到这属实是无能为力了，恳请大佬名师点拨

**Comment by yi-tong-yuan** [comment]
右边新品主题的文字和图片可以在yml文件中搜索新品主题即可修改，左边仅支持替换为技能块样式，具体配置方法官方文档中有提及https://docs.anheyu.com/advanced/#%E9%A6%96%E9%A1%B5%E6%8A%80%E8%83%BD%E7%82%B9%E9%85%8D%E7%BD%AE
